src/components/app.js

```diff
@@ -13,7 +13,8 @@ import { text } from 'd3-request';
 
 class App extends React.Component {
 
-  componentDidMount() {
+  async componentDidMount() {
+    await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
That seems like a bad fix. Is there a cleaner way to resolve this? Can we subscribe to an event from monaco?